utils/mail_service.py

```diff
@@ -0,0 +1,44 @@
+from typing import Tuple
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+from typing import Tuple
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from urllib.parse import urlencode
+
+def send_reset_password_email(email: str, reset_token: str) -> Tuple[str, str]:
+    sender_email = "user@example.com"
+    sender_password = "your_password" 
+
+    reset_password_url = f"http://127.0.0.1:8000/reset-password?{urlencode({'token': reset_token})}"
+
+    message = MIMEMultipart("alternative")
+    message["Subject"] = "Reset Password"
+    message["From"] = sender_email
+    message["To"] = email
+
+    text = f"Click on this link to reset your password: {reset_password_url}"
+    html = f"""\
+    <html>
+        <body>
+            <p>Click on this link to reset your password:</p>
+            <p><a href="{reset_password_url}">{reset_password_url}</a></p>
+        </body>
+    </html>
+    """
+
+    part1 = MIMEText(text, "plain")
+    part2 = MIMEText(html, "html")
+
+    message.attach(part1)
+    message.attach(part2)
+
+    with smtplib.SMTP("smtp.gmail.com", 587) as server:
+        server.starttls()
+        server.login(sender_email, sender_password)
+        server.sendmail(sender_email, email, message.as_string())
```

Comment:
Everything looks great but Will this send the emails? Is there anything we need to set up for this to work? I'm seeing some placeholder credentials on lines 13,14.

Reply:
Yes. They will send. For the mail you can use the organizational mail or any mail of your preference.

Reply:
I used Google and Outlook as my test mails. You'd have to set up an App password, as your generic password for your account